test/fixtures/config/program-flows.js

```diff
@@ -7,8 +7,15 @@ module.exports = {
       id: faker.datatype.uuid(),
       name: `${ faker.company.bsBuzz() } ${ faker.company.catchPhraseNoun() }`,
       details: faker.lorem.sentences(),
-      published: faker.datatype.boolean(),
-      behavior: faker.random.arrayElement(['standard', 'conditional']),
+      published_at: faker.random.arrayElement([faker.date.between(
+        dayjs().subtract(2, 'week').format(),
+        dayjs().subtract(1, 'week').format(),
+      ), null]),
+      archived_at: faker.random.arrayElement([faker.date.between(
+        dayjs().subtract(2, 'week').format(),
+        dayjs().subtract(1, 'week').format(),
+      ), null]),
```

Comment:
When generating fixture data, this randomly sets either a timestamp or `null` for `published_at` and `archived_at`. The same pattern is used across the programs, program flows and program actions fixtures.